common/src/main/java/com/wynntils/core/notifications/MessageContainer.java

```diff
@@ -9,21 +9,36 @@
 
 public class MessageContainer {
     protected TextRenderTask message;
+    private Integer iterations;
```

Comment:
This too.

Why not make this an int?

```suggestion
    private int iterations;
```

int is a primitive type.
Integer is an Object around int, which means Integet can be null, int can. We do not want this to be null.